src/components/ABVCalculator.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <form id="abvCalc" class="max-w-48 flex flex-col">
+    <label class="justify-center w-full flex flex-col my-4"
+      >Spirit ABV in %
+      <input
+        class="m-4 p-4 text-3xl text-center focus-within:outline-dashed focus:outline-green-500 focus:outline-4"
+        type="number"
+        v-model="spiritABV"
+        id="ABV"
+        placeholder="37.5"
+      />
+    </label>
+    <label class="justify-center w-full flex flex-col my-4"
+      >Spirit quantity in ml
+      <input
+        class="m-4 p-4 text-3xl text-center focus-within:outline-dashed focus:outline-green-500 focus:outline-4"
+        type="number"
+        v-model="spiritQuantity"
+        id="alcoholQuantity"
+        placeholder="50"
+      />
+    </label>
+    <label class="justify-center w-full flex flex-col my-4"
+      >Mixer quantity in ml
+      <input
+        class="m-4 p-4 text-3xl text-center focus-within:outline-dashed focus:outline-green-500 focus:outline-4"
+        type="number"
+        v-model="mixerQuantity"
+        id="mixerQuantity"
+        placeholder="330"
+      />
+    </label>
+  </form>
+
+  <div class="results">
+    <p>Your drink is <span :class="computedABVColourClasses" class="font-bold">{{ `${calculateTotalDrinkABV}% ABV` }}</span>
+    </p>
+    <p>
+      There is <span class="font-bold" :class="computeTotalAlcoholColourClasses">{{ calculateTotalAlcoholQuantity }}ml</span> of alcohol in your drink
+    </p>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      spiritABV: 37.5,
+      spiritQuantity: 50,
+      mixerQuantity: 330,
+    };
+  },
+  computed: {
+    abvToDecimal() {
+      return this.spiritABV / 100;
+    },
+    calculateTotalAlcoholQuantity() {
+      return this.spiritQuantity * this.abvToDecimal;
+    },
+    totalDrinkQuantity() {
+      return this.mixerQuantity + this.spiritQuantity;
+    },
+    calculateTotalDrinkABV() {

```

Comment:
Worth destructuring `totalDrinkQuantity`, `calculateTotalAlcoholQuantity`, and `totalDrinkQuantity`
eg:
```
const { totalDrinkQuantity, calculateTotalAlcoholQuantity, totalDrinkQuantity } = this;
```

Makes the code more readable and removes the dependency on `this`. Note it can't /  shouldn't be used with functions